static/app/components/replays/diff/replaySideBySideImageDiff.tsx

```diff
@@ -28,31 +29,24 @@ export function ReplaySideBySideImageDiff({leftOffsetMs, replay, rightOffsetMs}:
       </DiffHeader>
 
       <ReplayGrid>
-        <ReplayContextProvider
-          analyticsContext="replay_comparison_modal_left"
```

Comment:
do we lose analytics tracking by removing this?

Reply:
I see it's used for play/pause tracking: https://github.com/getsentry/sentry/blob/7b3f7bd38503664b33afa7aa05eabca098a2c66f/static/app/components/replays/replayContext.tsx#L494-L500

and also for fullscreen tracking. Neither are applicable to this hydration tool, so we should be good. But we'll need to include it in the new system as we refactor more.